router/dataplane_spec.gobra

```diff
@@ -20,9 +20,15 @@ import (
 	"net"
 	"hash"
 	sl "github.com/scionproto/scion/verification/utils/slices"
+	"github.com/scionproto/scion/verification/io"
+	
+	"github.com/scionproto/scion/verification/dependencies/encoding/binary"
+	"github.com/scionproto/scion/verification/utils/bitwise"
 	"github.com/scionproto/scion/pkg/scrypto"
 	"github.com/scionproto/scion/pkg/addr"
 )
+//"github.com/scionproto/scion/verification/definitions"
+
```

Comment:
let's drop these lines if they are not necessary

Reply:
Done.